CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+
+# Contributing
+
+Most of the time, if Webpack is not working correctly for you it is a simple configuration issue.
+
+If you are still having difficulty after looking over your configuration carefully, please post
+a question to [StackOverflow with the webpack tag](http://stackoverflow.com/tags/webpack). Questions
+that include your webpack.config.js and relevant files are more likely to receive responses.
+
+**If you have discovered a bug or have a feature suggestion, feel free to create an issue on Github.**
+
+If you have created your own loader/plugin please incude it on the relevant documentation pages:
+
+[List of loaders](http://webpack.github.io/docs/list-of-loaders.html)
+[List of plugins](http://webpack.github.io/docs/list-of-plugins.html)
+
+### Documentation
+
+Webpack is insanely feature rich and documentation is a huge time sink. We greatly appreciate any
+time spent fixing typos or clarifying sections in the documentation. 
+
+
+## Submitting Changes
+
+* Push to your fork and submit a pull request.
+* Wait for feedback. We may suggest some changes or improvements or alternatives.
+
+Some things that will increase the chance that your pull request is accepted:
+
+* Write tests.
+* Follow our coding style.
```

Comment:
There does not appear to be a rigorous coding guideline in place. However, there is an `.editorconfig` file that attempts to ensure whitespace formatting. Until there is something more defined around code style, telling people to use [editorconfig](http://editorconfig.org/) is the best bet, imo.